src/smart-components/role/role-permissions.js

```diff
@@ -141,45 +141,47 @@ const Permissions = () => {
   ];
 
   const toolbarButtons = () => [
-    ...[
-      // eslint-disable-next-line react/jsx-key
-      <Link to={`/roles/detail/${role.uuid}/role-add-permission`} key="role-add-permission" className="pf-m-visible-on-md">
-        <Button variant="primary" aria-label="Add Permission">
-          Add Permission
-        </Button>
-      </Link>,
-      {
-        label: 'Add Permission',
-        props: {
-          className: 'pf-m-hidden-on-md',
-        },
-        onClick: () => {
-          history.push(`/roles/detail/${role.uuid}/role-add-permission`);
-        },
+    window.insights.chrome.isBeta() && !role.system
+      ? [
+          // eslint-disable-next-line react/jsx-key
+          <Link to={`/roles/detail/${role.uuid}/role-add-permission`} key="role-add-permission" className="pf-m-visible-on-md">
+            <Button variant="primary" aria-label="Add Permission">
+              Add Permission
+            </Button>
+          </Link>,
+        ]
+      : [],
+    {
+      label: 'Add Permission',
+      props: {
+        className: 'pf-m-hidden-on-md',
       },
-      {
-        label: 'Remove',
-        props: {
-          isDisabled: !selectedPermissions.length > 0,
-        },
-        onClick: () => {
-          const multiplePermissionsSelected = selectedPermissions.length > 1;
-          internalDispatch({
-            type: INITIATE_REMOVE_PERMISSION,
-            confirmDelete: () => () => removePermissions([...selectedPermissions]),
-            deleteInfo: {
-              title: multiplePermissionsSelected ? 'Remove permissions?' : 'Remove permission?',
-              text: removeModalText(
-                multiplePermissionsSelected ? selectedPermissions.length : selectedPermissions[0].uuid,
-                role,
-                selectedPermissions.length > 1
-              ),
-              confirmButtonLabel: multiplePermissionsSelected ? 'Remove permissions' : 'Remove permission',
-            },
-          });
-        },
+      onClick: () => {
+        history.push(`/roles/detail/${role.uuid}/role-add-permission`);
+      },
+    },
+    {
+      label: 'Remove',
```

Comment:
Is this part also gated by `beta`? If not it should be.

Reply:
No only the first item in the array.

Reply:
It's still visible on small screens! Nice catch.

Reply:
Oh interesting, for some reason the dropdown is hidden on large screens.

Reply:
It's a bit weird. I think I'll have to look into that component. Because if you now have both the add permission and remove, you can't remove the selected permission on a large screen because the dropdown is just not there...

Reply:
So it won't affect prod but in prod beta users won't be able to remove permissions. Here's a question, should they be even able to remove permissions in prod? They can't add any so I would find it strange if users could remove them.

Reply:
Yeah, this is a part of https://issues.redhat.com/browse/RHCLOUD-11126 btw

Reply:
I think the gating for the "Remove" dropdown item should be the same as for the "Add permissions" button:
* only present for non-system roles in beta environments

Reply:
Right. It looks like missing CSS. The element is in the DOM. Only it has a `display: none` CSS property. After fixing it in the browser console it shows properly. I think I will try and fix it on Monday. And before we merge to prod.

> I think the gating for the "Remove" dropdown item should be the same as for the "Add permissions" button

Agreed.